src/tools/perf/perftest_context.h

```diff
@@ -50,6 +50,7 @@ struct perftest_context {
     unsigned                     num_cpus;
     unsigned                     cpus[MAX_CPUS];
     unsigned                     flags;
+    ucp_perf_daemon_info_t       dmn_info;
```

Comment:
why need to add this - isn't this already part of "params" ?

Reply:
There are 2 types of perftest params:
- global ones, configured once for all tests: remote server, port, AF, etc
- per-test params, configured per each individual test: test name, msg size, etc

Initially I put daemon params to the per-test params, which seems wrong. Logically we need a single daemon configured per batch on test (just like receiver server), and this config must not be allowed on per-test basis. That's why daemon config is now stored in the global params.
However, we still need to copy this config to per-test params, because at runtime perftest needs this information in order to connect to the daemon. I didn't find a clean way to access global properties from the perftest `ucp_perf_setup`. So I thought that an easiest option is just to copy this struct from global-params to per-test-params.

Reply:
I reflected a bit more on that, and I think you are right, I will remove this duplication.